test/remittance.js

```diff
@@ -4,7 +4,267 @@ const Remittance = artifacts.require("./Remittance.sol");
 contract('Remittance', async accounts => {
 
     const { toBN } = web3.utils;
+    const concatPassword = "password1" + "password2";
+    const hash = await web3.utils.soliditySha3(concatPassword);
 
+    const getGasCost = async txObj => {
```

Comment:
Would be good to use it :)

Reply:
Sorry. I have enabled pre-commit analysis to pick up these basis mistakes.